src/ResourceManager/Sql/Commands.Sql/Elastic Pools/Cmdlet/SetAzureSqlElasticPool.cs

```diff
@@ -142,6 +142,13 @@ public class SetAzureSqlElasticPool : AzureSqlElasticPoolCmdletBase
             HelpMessage = "The zone redundancy to associate with the Azure SQL Elastic Pool")]
         public SwitchParameter ZoneRedundant { get; set; }
 
+        /// <summary>
+        /// Gets or sets the license type for the Azure Sql database
+        /// </summary>
+        [Parameter(Mandatory = false, HelpMessage = "The license type for the Azure Sql database")]
+        [PSArgumentCompleter("LicenseIncluded", "BasePrice")]
```

Comment:
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/SqlManagement/Management.Sql/Generated/Models/ElasticPoolLicenseType.cs